rholang/src/main/scala/coop/rchain/rholang/interpreter/Reduce.scala

```diff
@@ -192,18 +192,22 @@ object Reduce {
           case _ => s.unit
         }
 
-      def mkJob[A](terms: Seq[A], handler: (A, Blake2b512Random) => M[Unit]): M[List[Unit]] =
+      def mkJob[A](
+          terms: Seq[A],
+          handler: (A, Blake2b512Random) => M[Unit],
+          startIdx: Int
```

Comment:
How about making `startIdx` the first parameter, for some nice alignment and faster-spotted copy-paste errors?

Reply:
outdated